### PR TITLE
chore(package.json/package-lock.json):update notifications-panel version

### PR DESCRIPTION
**Pull request by Enzo707**
This PR is for updating the notifications-panel version we use in NotificationCenter to 0.0.7

### CODE REVIEW

**Comment by zeusongit** [comment]
@Enzo707 Is notifications-panel 0.0.7 not published yet?

**Comment by Enzo707** [comment]
@avidit 

> @Enzo707 Is notifications-panel 0.0.7 not published yet? The build failed because the dependencies could not be resolved

Yes, it's already published [0.0.7](https://www.npmjs.com/package/@dynamods/notifications-panel/v/0.0.7)

**Comment by zeusongit** [comment]
Seems like a huge diff in package-lock file for a single package update. What else did we update?

**Comment by QilongTang** [comment]
Seems to be the same problem that @Enzo707 You want to disconnect from VPN when modifying the package lock, otherwise, it will try to replace all the dependencies to point to ADSK npm artifactory

**Comment by Enzo707** [comment]
> Seems like a huge diff in package-lock file for a single package update. What else did we update?

I've just updated the notifications-panel version but probably running "npm install" locally has overwritten it.

**Comment by QilongTang** [comment]
> > Seems like a huge diff in package-lock file for a single package update. What else did we update?
> 
> I've just updated the notifications-panel version but probably running "npm install" locally has overwritten it.

yeah I think usually the trick is edit the lock file not on VPN or just manually only on the version number

**Comment by Enzo707** [comment]
> > > Seems like a huge diff in package-lock file for a single package update. What else did we update?
> > 
> > 
> > I've just updated the notifications-panel version but probably running "npm install" locally has overwritten it.
> 
> yeah I think usually the trick is edit the lock file not on VPN or just manually only on the version number

@QilongTang I haven't  used the VPN at the time...  anyway I could try just modifying the version number to see if it works

File: package-lock.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "@babel/core": "^7.18.6",
         "@babel/preset-env": "^7.18.6",
         "@babel/preset-react": "^7.18.6",
-        "@dynamods/notifications-panel": "^0.0.6",
+        "@dynamods/notifications-panel": "^0.0.7",
```
Comment (QilongTang) [inline package-lock.json:19]: I think only change here should work

File: package-lock.json [hunk 2]
```diff
@@ -1861,8 +1861,8 @@
       }
     },
     "node_modules/@dynamods/notifications-panel": {
-      "version": "0.0.6",
-      "resolved": "https://registry.npmjs.org/@dynamods/notifications-panel/-/notifications-panel-0.0.6.tgz",
+      "version": "0.0.7",
+      "resolved": "https://registry.npmjs.org/@dynamods/notifications-panel/-/notifications-panel-0.0.7.tgz",
```
Comment (QilongTang) [inline package-lock.json:1865]: This would cause the sha512 mismatch with the version

Comment (Enzo707) [reply package-lock.json]: @QilongTang I've rolled back my changes according to your suggestion but the build still failed, unfortunately

Comment (Enzo707) [reply package-lock.json:1873]: @QilongTang  @zeusongit Just for the record 
`npm i -g npm
rm -rf node_modules/
npm cache clear --force 
npm i`
solved the checksum integrity issue

**Comment by zeusongit** [comment]
@Enzo707 Build passed, but is there a reason you are force-pushing the changes each time? These are usually not recommended, since they will overwrite any conflicts, if any.

**Comment by Enzo707** [comment]
> @Enzo707 Build passed, but is there a reason you are force-pushing the changes each time? These are usually not recommended, since they will overwrite any conflicts, if any.

I know it's not recommended but as I was making minor changes on the package/package-lock I used commit amend to avoid creating multiple commits for the same thing. If you prefer I could skip this in the future.

**Pull request closed without merging**